docs/en/ingest-management/fleet/fleet-server.asciidoc

```diff
@@ -48,12 +48,11 @@ a self-managed cluster).
 bugfix releases)
 ** {kib} should be on the same minor version as {es}.
 
-//TODO: Uncomment this section when ECE 2.10 is available
-//* {ece} 2.10 or later
-//** Requires additional wildcard domains and certificates (which normally only
-//cover *.cname, not ..cname). This enables us to provide the URL for
-//{fleet-server} of `https://.fleet.`
-//** The deployment template must contain a slider for APM & Fleet.
+* {ece} 2.10 or later
```

Comment:
Only the Fleet Server in the APM & Fleet slider requires ECE 2.10 or higher. If you have ECE 2.9 or lower you can use a self-hosted Fleet Server.